### PR TITLE
wishlist done

### PR DESCRIPTION
**Pull request by sakshamAK**
image

### CODE REVIEW

File: wishlist/style.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+}
+
+.menu-list-items {
+    padding: 1rem;
+}
+
+a {
+    color: var(--font-primary);
+    text-decoration: none;
+}
```
Comment (adidoshi) [inline wishlist/style.css:21]: Just a suggestion, if we can have `a` tag styling in base css so that it can be used everywhere.

**Review by adidoshi — APPROVED** [review]
Also if the dismiss button background is intentional - then good, or maybe you can change the bg-color, rest looks great. 👍

File: wishlist/style.css (hunk 1 above)
Comment (sakshamAK) [reply wishlist/style.css:21]: Sure, thanks 😄 
I will do the refactoring all together later

**Comment by sakshamAK** [comment]
> Also if the dismiss button background is intentional - then good, or maybe you can change the bg-color, rest looks great. 👍

It was intentional though but what color should I go for as an alternate?

**Comment by poornima-alugubelly** [comment]
not sure if u have handled responsiveness part , because I observed that the card are overflowing might want to check on that 
image

**Review by poornima-alugubelly — APPROVED** [review]
great work Saksham everything looks good!

**Pull request merged**